pipelines/release.yml

```diff
@@ -1341,6 +1375,15 @@ resources:
     repository: concourse/concourse-bosh-release
     regexp: ((release_minor)).*
 
+- name: pivnet-bosh-release
+  type: pivnet
+  icon: chart-timeline-variant
+  source:
+    api_token: ((pivnet-token))
+    product_slug: p-concourse
+    copy_metadata: true
+    endpoint: https://network.pivotal.io
```

Comment:
Not sure if you know, but I imagine that this endpoint will change at some point to not be under `pivotal.io` - might it make sense to not define the `endpoint`? The default is already `https://network.pivotal.io`, and I speculate they'll update that if Tanzu network gets moved elsewhere

Reply:
Though `https://network.pivotal.io` may stick around for a while, I'm not sure

Reply:
nice, I didn't realize it had a default!